platforms-serverless/firebase-functions/prepare.sh

```diff
@@ -2,4 +2,4 @@
 
 set -eux
 
-npm install -g firebase-tools@9.14.0
+pnpm install -g firebase-tools@11.25.2
```

Comment:
we should probably try to do that via package.json so this can automatically be updated